packages/synapse-react-client/src/lib/containers/ConfirmationDialog.tsx

```diff
@@ -2,21 +2,19 @@ import { Button, ButtonProps } from '@mui/material'
 import React from 'react'
 import { DialogBase, DialogBaseProps } from './DialogBase'
 
-export type ConfirmationDialogProps = DialogBaseProps & {
+export type ConfirmationButtonsProps = {
+  onConfirm: () => void
+  onCancel: DialogBaseProps['onCancel']
```

Comment:
Pull the confirmation buttons into their own component, so they can be reused in the ForumThreadEditor